exercises/follow_line/web-template/assets/ws_code.js

```diff
@@ -53,12 +72,32 @@ function stopCode(){
     websocket_code.send(stop_code);
 }
 
+// Function to save the code locally
+function saveCodeLocally(){
+	var python_code = editor.getValue();
+	var blob = new Blob([python_code], {type: "text/plain; charset=utf-8"});
+	if (window.navigator.msSaveOROpenBlob)
+		window.navigator.msSaveOrOpenBlob(blob, "follow_line.py");
```

Comment:
The file name to which we are saving should be some common name, for instance: `exercise.py` or `academy.py`